### PR TITLE
[15.0][MIG] stock_split_picking

### PR DESCRIPTION
**Pull request by BernatPForgeFlow**

### CODE REVIEW

**Comment by BernatPForgeFlow** [comment]
@ForgeFlow
@AaronHForgeFlow 
@LoisRForgeFlow

**Review by AaronHForgeFlow — APPROVED** [review]
Tested successfully, the 3 splits methods. Working the same as in previous versions.

File: stock_split_picking/wizards/stock_split_picking.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
                         attrs="{'invisible': [('mode', '!=', 'selection')], 'required': [('mode', '=', 'selection')]}"
                         domain="[('picking_id', 'in', picking_ids)]"
                     >
-                        <tree string="Moves" create="true" edit="true" delete="true">
+                        <tree name="Moves" create="true" edit="true" delete="true">
```
Comment (LoisRForgeFlow) [inline stock_split_picking/wizards/stock_split_picking.xml]: remove `string="moves"`, it is derprecated.

**Review by LoisRForgeFlow — APPROVED** [review]

**Pull request merged**